### DISCUSSION TITLE
ssh-keygen -t rsa -b 4096 -C no responde

### DISCUSSION THREAD

**Discussion by EricVizard** [community/community#33017] [Other Feature Feedback, Questions, & Ideas]
Hola, estaba ejecutando el comando :  ssh-keygen -t rsa -b 4096 -C "Micorreo@..."

Sin embargo no responde nada y la consola se queda colgada, cuando intento cerrar me aparece

![image](https://user-images.githubusercontent.com/111129566/190108283-e22f0ae9-5185-4961-a83a-7270e182c8b1.png)

Su ayuda como pueda hacer que responda ya que ese proceso se queda ejecutando.

**Comment by airtower-luna** [comment]
Generating a key may take a while depending on the source of entropy (randomness) used. Try doing some other stuff on the same computer while you wait, mouse and keyboard events are often used to feed the random generator. If that doesn't help please say a bit more about the system you're using, `mintty` looks a bit like you might be using a remote system.

**Reply by EricVizard** [reply]
Espere 1 día y nunca avanzo. Cerre y volvi a abrir y me aparecieron 3 comandos en espera :(  ... Es pc local, no es remoto.
Ya instale y desinstale GIT sin resultado, al invocar el comando ssh-keygen -t rsa -b 4096 -C "Mi_correo" se queda la pantalla sin avanzar.

**Reply by airtower-luna** [reply]
Okay, it definitely shouldn't take a day, I was thinking of a couple minutes. :sweat_smile: 

I don't know what might be causing that problem, but one thing you could try is to generate a different type of key, e.g. ED25519 instead of RSA. Maybe the generator for that doesn't have the same problem.
